js/jquery.terminal.d.ts

```diff
@@ -358,9 +359,10 @@ type CmdOptions = {
     tabindex?: number;
     mobileDelete?: boolean;
 }
-type CmdOption = "mask" | "caseSensitiveSearch" | "historySize" |  "prompt" | "enabled" |
-    "history" | "tabs" | "onPositionChange" |  "clickTimeout" |  "holdTimeout" | "onPaste" |
-    "holdRepeatTimeout" |  "repeatTimeoutKeys" | "width" | "historyFilter" | "commands" |
+
+type CmdOption = "mask" | "caseSensitiveSearch" | "historySize" | "prompt" | "enabled" |
+    "history" | "tabs" | "onPositionChange" | "clickTimeout" | "holdTimeout" | "onPaste" |
+    "holdRepeatTimeout" | "repeatTimeoutKeys" | "width" | "historyFilter" | "commands" |
     "char_width" | "onCommandChange" | "name" | "keypress" | "keydown" | "mobileDelete";
```

Comment:
What did you change here?

Reply:
Linting. There’re some `|` operators having two spaces before and/or after.